collection-scripts/gather_network_logs

```diff
@@ -73,6 +74,48 @@ function gather_scale_data {
   echo pods amount: $(oc get pods -A | wc -l) >> "${NETWORK_LOG_PATH}"/cluster_scale & PIDS+=($!)
 }
 
+function gather_metallb_data {
+  echo "INFO: Gathering metallb data"
+  declare -a FILES_TO_GATHER=("frr.conf" "frr.log" "daemons" "vtysh.conf")
+  declare -a FRR_CONTAINERS=("frr" "reloader" "frr-metrics")
+  SPEAKER_PODS="${@:-$(oc -n ${METALLB_NS} get pods -l component=speaker -o jsonpath='{.items[*].metadata.name}')}"
```

Comment:
This may change soon. 
cc @sabinaaledort

Reply:
This may change as a result of what?

Reply:
the speaker label is going to change:
https://github.com/metallb/metallb-operator/pull/124